pkg/util/paramtable/service_param.go

```diff
@@ -1101,6 +1101,7 @@ type MinioConfig struct {
 	Port               ParamItem `refreshable:"false"`
 	AccessKeyID        ParamItem `refreshable:"false"`
 	SecretAccessKey    ParamItem `refreshable:"false"`
+	SignatureVersion   ParamItem `refreshable:"false"`
```

Comment:
Thank you for contributing to the code! Please refer to the other configuration items and add the initialization logic for the configuration items. If it's convenient for you to add some unit tests, you'll find that the missing logic here will cause the configuration to fail to take effect.

Reply:
init logic added in new patch. It's hard to write ut for this pr, <https://min.io/docs/kes/tutorials/configuration/> here, i cannot find minio support old sign method config.